src/components/slider/slider.scss

```diff
@@ -0,0 +1,120 @@
+@import 'default-theme';
+
+$slider-size: 20px !default;
+$slider-min-size: 128px !default;
+
+$slider-track-height: 2px !default;
+$slider-thumb-width: 20px !default;
+$slider-thumb-height: $slider-thumb-width !default;
+
+$slider-thumb-default-scale: 0.7 !default;
+$slider-thumb-focus-scale: 1 !default;
+
+// TODO(iveysaur): Find an implementation to hide the track under a disabled thumb.
+$slider-off-color: rgba(black, 0.26);
+$slider-focused-color: rgba(black, 0.38);
+$slider-disabled-color: rgba(black, 0.26);
+
+@mixin slider-thumb-position($width: $slider-thumb-width, $height: $slider-thumb-height) {
+  position: absolute;
+  top: ($slider-size / 2) - ($height / 2);
+  width: $width;
+  height: $height;
+  border-radius: max($width, $height);
+}
+
+md-slider {
+  height: $slider-size;
+  min-width: $slider-min-size;
+  position: relative;
+  padding: 0;
+  display: inline-block;
+  flex-direction: row;
+  outline: none;
+  vertical-align: middle;
+}
+
+md-slider *,
+md-slider *::after {
+  box-sizing: border-box;
+}
+
+.md-slider-wrapper {
+  width: 100%;
+  height: 100%;
+  padding-left: 8px;
+  padding-right: 8px;
+}
+
+.md-slider-container {
+  position: relative;
+}
+
+.md-slider-track-container {
+  width: 100%;
+  position: absolute;
+  top: ($slider-size / 2) - ($slider-track-height) / 2;
+  height: $slider-track-height;
+}
+
+.md-slider-track {
+  position: absolute;
+  left: 0;
+  right: 0;
+  height: 100%;
+  background-color: $slider-off-color;
+}
+
+.md-slider-track-fill {
+  transition: all 400ms cubic-bezier(0.25, 0.8, 0.25, 1);
```

Comment:
I think it might make sense to either remove `all` (which should work the same way) or specify `transition-duration` and `transition-timing-function` separately to make it more obvious what is going on.